### ISSUE TITLE
smooth lighting bug

### ISSUE THREAD

**Issue by ivoryh2o** [ate47/Xray#21]
I'm currently playing on the latest version of fabric, there is a bug with the smooth lighting that makes it look very blocky, like it is turned off. I tried playing without this mod installed and the issue is not present.

**Comment by ZYFDroid** [comment]
How to fix this bug in fabric 1.17.1
Modify `MixinMinecraftClient.java` like this 
```
if (XrayMain.getMod().getSelectedMode() != null) {
	if(XrayMain.getMod().getSelectedMode().isEnabled()) {
		ci.setReturnValue(false);
		ci.cancel();
	}
}
```

**Comment by ZYFDroid** [comment]
It seems the mod always disables AmbientOcclusion (since getSelectedMode never returns null, no matter whether mode is enabled)
